modules/gui/surveillance.lua

```diff
@@ -0,0 +1,167 @@
+---- module surveillance
+-- @addon surveillance
+
+local Gui = require 'expcore.gui' --- @dep expcore.gui
+local Roles = require 'expcore.roles' --- @dep expcore.roles
+local Event = require 'utils.event' --- @dep utils.event
+
+local cctv_container
+
+local cctv_player =
+Gui.element(function(name, parent, player_list)
+    return parent.add{
+        name = name,
+        type = 'drop-down',
+        items = player_list,
+        selected_index = #player_list > 0 and 1
+    }
+end)
+
+local cctv_type =
+Gui.element{
+    type = 'drop-down',
+    name = 'cctv_status',
+    items = {'Enable', 'Disable'},
+    selected_index = 1
+}:style{
+    width = 80
+}:on_selection_changed(function(_, element, _)
+    if element.selected_index == 1 then
+        element.parent.parent.parent.cctv_display.visible = true
+    else
+        element.parent.parent.parent.cctv_display.visible = false
+    end
+end)
+
+local cctv_status =
+Gui.element{
+    type = 'drop-down',
+    name = 'cctv_status',
+    items = {'Player', 'Static'},
+    selected_index = 1
+}:style{
+    width = 80
+}
+
+local cctv_location =
+Gui.element{
+    type = 'button',
+    caption = 'set'
+}:style{
+    width = 40
+}:on_click(function(player, element, _)
+    element.parent.parent.cctv_display.position = player.position
```

Comment:
Very simple correction here, this one is probably on me when I changed the GUI structure.
```suggestion
    element.parent.parent.parent.cctv_display.position = player.position
```